paymentsheet-example/src/androidTest/java/com/stripe/android/test/core/PlaygroundTestDriver.kt

```diff
@@ -563,11 +563,11 @@ internal class PlaygroundTestDriver(
                         .size == 1
                 }
 
-                composeTestRule.waitUntil {
-                    composeTestRule.onAllNodesWithText("Country or region")
-                        .fetchSemanticsNodes()
-                        .size == 1
-                }
+//                composeTestRule.waitUntil {
```

Comment:
should this be deleted now instead of commented out?

Reply:
This must have been an outdated diff. I was trying to debug what was going on, but it's back now!